scripts/zones/Temenos/bcnms/temenos_northern_tower.lua

```diff
@@ -2,38 +2,47 @@
 -- Area: Temenos
 -- Name: Temenos Northern Tower
 -----------------------------------
-require("scripts/globals/limbus");
+require("scripts/globals/limbus")
 require("scripts/globals/battlefield")
-require("scripts/globals/keyitems");
+require("scripts/globals/keyitems")
+local ID = require("scripts/zones/Temenos/IDs")
+
+function onBattlefieldInitialise(battlefield)
+    battlefield:setLocalVar("loot", 1)
+    battlefield:setLocalVar("randomF1", math.random(1,6))
+    SetServerVariable("[Temenos_Northern_Tower]Time", battlefield:getTimeLimit()/60)
+    tpz.limbus.handleDoors(battlefield)
+    tpz.limbus.setupArmouryCrates(battlefield:getID())
+    DespawnMob(ID.mob.TEMENOS_N_MOB[3]+3)
+    DespawnMob(ID.mob.TEMENOS_N_MOB[4]+3)
+    DespawnMob(ID.mob.TEMENOS_N_MOB[6]+4)
+    DespawnMob(ID.mob.TEMENOS_N_MOB[6]+8)
+    DespawnMob(ID.mob.TEMENOS_N_MOB[6]+12)
```

Comment:
What are these despawns for?

Reply:
I think the issue was I was getting navmesh errors if I don't spawn the pets at the start, and they'll be spawned before they should be (DRG pets) if I don't despawn them.